### ISSUE TITLE
[BUG] npm v8 install is 9 times slower than npm 6.14.13 - reproducible

### ISSUE THREAD

**Issue by patoi** [npm/cli#3886]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

npm  v6.14.13 install much faster than npm v8 and v6.14.15 (comes with latest Node.js LTS)

### Expected Behavior

Near same runtime.

### Steps To Reproduce

```
npm init -y

nvm use v14.17.0
npm -v
6.14.13
time npm i -D carbon-components-svelte 
npm i -D carbon-components-svelte  **5,14s user** 10,40s system 181% cpu 8,575 total

nvm use v16
npm -v
8.0.0
rm -rf ./node_modules
time npm i -D carbon-components-svelte
npm i -D carbon-components-svelte  **46,93s user** 11,72s system 116% cpu 50,436 total

nvm use v14.18.1
npm -v
6.14.15
rm -rf ./node_modules
time npm i -D carbon-components-svelte
npm i -D carbon-components-svelte  **252,60s user** 17,14s system 103% cpu 4:19,55 total
```

### Environment

- OS: MacOS
- Node: v14.17.0, v16.11.1 and v14.18.1
- npm: v6.14.13, v8.0.0 and v6.14.15

**Comment by patoi** [comment]
```
npm -v
8.1.0
time npm i -D carbon-components-svelte
npm i -D carbon-components-svelte  **47,87s user** 11,21s system 119% cpu 49,417 total
```

**Comment by patoi** [comment]
```
npm -v
8.1.1
time npm i -D carbon-components-svelte
npm i -D carbon-components-svelte  **49,95s user** 11,49s system 116% cpu 52,587 total
```

**Comment by patoi** [comment]
```
npm -v
8.1.3
time npm i -D carbon-components-svelte
npm i -D carbon-components-svelte  **0,53s user** 0,20s system 45% cpu 1,617 total
```

**Comment by patoi** [comment]
Closed, because npm 8.1.3 and carbon-components-svelte 0.49.0 performance is great.
